src/test/java/org/slieb/throwables/BiConsumerWithThrowableRethrowTest.java

```diff
@@ -0,0 +1,34 @@
+package org.slieb.throwables;
+import org.junit.Test;
+import java.io.IOException;
+import static org.slieb.throwables.BiConsumerWithThrowable.rethrowBiConsumer;
+@java.lang.SuppressWarnings({"CodeBlock2Expr"})
+public class BiConsumerWithThrowableRethrowTest {
+
+
+  @Test
+  public void testThrowCheckedException() {
+    IOException expected = new IOException("EXPECTED ERROR");
+    try {
+      rethrowBiConsumer((v1, v2) -> {
+        throw expected;
+      }).accept(null, null);
+    } catch (IOException actual) {
+      org.junit.Assert.assertEquals(expected, actual);
+    }
+  }
```

Comment:
I wrote it this way to check that the exact exception is rethrown and nothing is wrapped in a new exception, but if you prefer it with `@Test(expected = IOException.class)` i can change that.